quiz.js

```diff
@@ -37,7 +50,45 @@ db.dropDatabase();
 
 // your code begins here
 
-
+db.houses.find({ name: 'Arryn'});
+
+db.houses.update({ name: 'Arryn'
+}, {
+  $push: {
+    people: {
+      name: 'Jon Arryn'
+    }
+  }
+});
+
+db.houses.find({ name: 'Stark'})
+
+db.houses.update({ name: 'Stark'
+}, {
+  $push: {
+    people: {
+      name: 'Ned Stark',
+      name: 'Arya Stark',
+      name: 'Sansa Stark'
+    }
+  }
```

Comment:
Unfortunately, this won't work. Because $push is expecting a single value, in the form of either a string or an object, what you've done here is try to give it a single object (with three identical keys) and made that object the _first element_ of `people`. If you were to push other things to `people`, they would live outside this object, like so:
```
db.houses.update({name : 'Stark' }, { $push : { people : { name : 'Bran Stark' } } })
....
{ "_id" : ObjectId("............................................"), "name" : "Stark", "motto" : "Winter is Coming", "people" : [ {...}, { name : 'Bran Stark' } ] }
```

What's more, because all three of those keys are the same, you've actually been redefining 'name' each time, so only the final value, 'Sansa Stark', gets preserved.

The right way to write this would be with the $each operator, as follows:
```
db.houses.update({ name : 'Stark' }, { $push : { people : 
  { $each : [
    { name : 'Ned Stark' },
    { name : 'Arya Stark' },
    { name : 'Sansa Stark' }
  ] } 
} });
```